### PR TITLE
Added custom command, added couple of fixes to factories

### PR DESCRIPTION
**Pull request by pazuzu-ua**
DO NOT MERGE UNTIL CHECKED

Added new custom command for populating db and for easier migration.

Usage:
- To add elements:
`python manage.py populate`
- To clear database before adding new items (IMPORTANT: it will flush your db, it will delete all entries, than it will perform migration - good for full reset of repo)
`python manage.py populate --force`
- To create superuser:
`python manage.py populate --superuser`
- Or everything at the same time:
`python manage.py populate --force --superuser`

### CODE REVIEW

**Review by Misha86 — APPROVED** [review]
Approved

**Review by morento101 — APPROVED** [review]
Awesome!!!

File: beauty/api/management/commands/populate.py [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+                call_command("migrate")
+            if options["superuser"]:
+                call_command("createsuperuser")
+            CustomUserFactory.create_batch(15, is_active=True)
+            OrderFactory.create_batch(15)
+            ReviewFactory.create_batch(
+                10,
+                from_user=CustomUser.objects.get(pk=randint(0, 30)),
+                to_user=CustomUser.objects.get(pk=randint(0, 30)),
+            )
```
Comment (therobotisnotatoy) [inline beauty/api/management/commands/populate.py]: And if both random pk are the same? I believe it was told that user can not add review to itself.

**Review by therobotisnotatoy — CHANGES_REQUESTED** [review]

File: beauty/api/management/commands/populate.py (hunk 1 above)
Comment (morento101) [reply beauty/api/management/commands/populate.py]: Will adding CustomUser SubFactories to ReviewFactory resolve this issue?

Comment (pazuzu-ua) [reply beauty/api/management/commands/populate.py]: In order to not to make extra people, rewrote it with choices.

**Review by therobotisnotatoy — APPROVED** [review]

**Pull request merged**